src/proto/mgmt/pool.proto

```diff
@@ -173,14 +173,19 @@ message PoolQueryReq {
 	bool include_disabled_ranks = 5; // True if the list of disabled ranks shall be returned
 }
 
+enum StorageMediaType {
+	SCM = 0;
+	NVME = 1;
+}
+
 // StorageUsageStats represents usage statistics for a storage subsystem.
 message StorageUsageStats {
 	uint64 total = 1;
 	uint64 free = 2;
 	uint64 min = 3;
 	uint64 max = 4;
 	uint64 mean = 5;
-	uint32 media_type = 6;
+	StorageMediaType media_type = 6;
```

Comment:
@mjmac does this change have implications for backwards compatibility? Enums are just ints, after all, but I'm not sure how fussy protobuf is about typing.